src/sentry/eventstream/snuba.py

```diff
@@ -1,50 +1,10 @@
 from __future__ import absolute_import
 
-from sentry.eventstream.base import EventStream
+from sentry.eventstream.kafka import KafkaEventStream
+from sentry.eventstream.kafka.backend import EVENT_PROTOCOL_VERSION
 from sentry.utils import snuba
 
 
-class SnubaEventStream(EventStream):
-    def insert(self, group, event, is_new, is_sample, is_regression,
-               is_new_group_environment, primary_hash, skip_consume=False):
-        snuba.insert_raw([{
-            'group_id': event.group_id,
-            'event_id': event.event_id,
-            'project_id': event.project_id,
-            # TODO(mitsuhiko): We do not want to send this incorrect
-            # message but this is what snuba needs at the moment.
-            'message': event.message,
-            'platform': event.platform,
-            'datetime': event.datetime,
-            'data': dict(event.data.items()),
-            'primary_hash': primary_hash,
-        }])
-        super(SnubaEventStream, self).insert(
-            group, event, is_new, is_sample,
-            is_regression, is_new_group_environment,
-            primary_hash, skip_consume
-        )
-
-    def start_delete_groups(self, project_id, group_ids):
-        pass
-
-    def end_delete_groups(self, state):
-        pass
-
-    def start_merge(self, project_id, previous_group_ids, new_group_id):
-        pass
-
-    def end_merge(self, state):
-        pass
-
-    def start_unmerge(self, project_id, hashes, previous_group_id, new_group_id):
-        pass
-
-    def end_unmerge(self, state):
-        pass
-
-    def start_delete_tag(self, project_id, tag):
-        pass
-
-    def end_delete_tag(self, state):
-        pass
+class SnubaEventStream(KafkaEventStream):
+    def _send(self, project_id, _type, extra_data=(), asynchronous=True):
+        snuba.test_eventstream((EVENT_PROTOCOL_VERSION, _type) + extra_data)
```

Comment:
Is this format change going to break for people using this for local development?

Reply:
There is a corresponding Snuba PR and they will need to update Snuba for this to work. Otherwise it shouldn't, I did this specifically for them -- I tried out merge locally etc. Or am I missing something about the question?

Reply:
https://github.com/getsentry/snuba/pull/274

Reply:
(I hadn't checked out the other side of this when I wrote the comment.) We should give people a heads up that when they pull changes after this is merged that they need to do the same for Snuba at the same time, since trying to debug this could be frustrating unless you knew the underlying protocol changed and I don't think most people are preemptively updating their Docker images.